index.php

```diff
@@ -44,6 +44,21 @@ function getDirectory($path, $allowed_filenames, $level = 0)
     return $retval;
 }
 
+function showGrades($ruser, $csvFile)
+{
+    # Show lines from the csvFile where cwlid=$ruser
+    $retVal = "";
+    $csv = array_map("str_getcsv", file($csvFile, FILE_SKIP_EMPTY_LINES));
+    $keys = array_shift($csv);
+    foreach ($csv as $i=>$row) {
+        $csv[$i] = array_combine($keys, $row);
+        if (array_key_exists('cwlid', $csv[$i]) and $csv[$i]['cwlid'] == $ruser) {
```

Comment:
This might not need to change but would make me feel better and easier to read.

`if (!empty($csv[$i]['cwlid']) && $csv[$i]['cwlid'] === $ruser) {`

Reply:
added thanks!